x-pack/plugins/security_solution/public/management/components/artifact_list_page/artifact_list_page.tsx

```diff
@@ -352,7 +352,7 @@ export const ArtifactListPage = memo<ArtifactListPageProps>(
               ItemComponent={ArtifactEntryCard}
               itemComponentProps={handleCardProps}
               onChange={handlePaginationChange}
-              error={error}
+              error={error as React.ReactNode}
```

Comment:
Perhaps consider tightening the type for the component props, as it seems that error on this side isn’t intended to be a ReactNode.